paloalto/package.yaml

```diff
@@ -0,0 +1,75 @@
+        @name = "paloalto.ip-tags"
+      } else if ",USER-ID," in content {
+        legacy "parse content csv --header 'future_use1,receive_time,serial_number,type,log_subtype,version,generated_time,vsys,src_ip,user,datasource_name,event_id,repeat_count,timeout_threshold,src_port,dest_port,datasource,datasource_type,sequence_number,action_flags,devicegroup_level1,devicegroup_level2,devicegroup_level3,devicegroup_level4,vsys_name,dvc_name,vsys_id,factor_type,factor_completion_time,factor_number,ugflags,userbysource,high_res_timestamp'"
+        @name = "paloalto.user-id"
+      } else if ",DECRYPTION," in content {
+        legacy "parse content csv --header 'future_use1,receive_time,serial_number,type,log_subtype,version,generated_time,src_ip,dest_ip,src_translated_ip,dest_translated_ip,rule,src_user,dest_user,app,vsys,src_zone,dest_zone,src_interface,dest_interface,log_forwarding_profile,start_time,session_id,repeat_count,src_port,dest_port,src_translated_port,dest_translated_port,flags,ip_protocol,action,tunnel_id,future_use2,future_use3,src_vm_uuid,dest_vm_uuid,uuid_rule,stage_client_firewall,stage_firewall_client,tls_version,key_exchange_algorithm,encryption_algorithm,hash_algorithm,policy_name,elliptic_curve,error_index,root_status,chain_status,proxy_type,cert_serial_number,fingerprint,cert_start_time,cert_end_time,cert_version,cert_size,cn_length,issuer_cn_length,root_cn_length,sni_length,cert_flags,subject_cn,issuer_subject_cn,root_subject_cn,server_name,error,container_id,pod_namespace,pod_name,src_edl,dest_edl,src_dag,dest_dag,timestamp,src_dvc_category,src_dvc_profile,src_dvc_model,src_dvc_vendor,src_dvc_os,src_dvc_os_version,src_name,src_mac,dest_dvc_category,dest_dvc_profile,dest_dvc_model,dest_dvc_vendor,dest_dvc_os,dest_dvc_os_version,dest_name,dest_mac,sequence_number,action_flags,devicegroup_level1,devicegroup_level2,devicegroup_level3,devicegroup_level4,vsys_name,dvc_name,vsys_id,app_subcategory,app_category,app_technology,app_risk,app_characteristic,app_container,app_tunneled,app_saas,app_sanctioned'"
+        @name = "paloalto.decryption"
+      } else if ",AUTHENTICATION," in content {
+        legacy "parse content csv --header 'future_use1,receive_time,serial_number,type,log_subtype,version,generated_time,virtual_system,src_ip,user,normalize_user,object,authentication_policy,repeat_count,authentication_id,vendor,log_action,server_profile,description,client_type,event_type,factor_number,sequence_number,action_flags,dvc_group_hierarchy1,dvc_group_hierarchy2,dvc_group_hierarchy3,dvc_group_hierarchy4,virtual_system_name,dvc_name,virtual_system_id,authentication_protocol,uuid_for_rule,high_resolution_timestamp,src_dvc_category,src_dvc_profile,src_dvc_model,src_dvc_vendor,src_dvc_os_family,src_dvc_os_version,src_hostname,src_mac_address,region,future_use3,user_agent,session_id'"
+        @name = "paloalto.authentication"
+      } else {
+        @name = "paloalto.unknown"
+      }
+      publish "paloalto"
```

Comment:
Ideally, mention that the pipeline publishes to a topic called `paloalto` in the pipeline's description.